src/adic_space.lean

```diff
@@ -76,16 +80,16 @@ A morphism in 𝒞 is a map of top spaces, an f-map of presheaves, such that the
 map on the stalks pulls one valuation back to the other.
 -/
 
-instance presheaf_of_rings.comm_ring {X : Type*} [topological_space X]
+instance presheaf_of_rings.comm_ring {X : Type u} [topological_space X]
   (F : presheaf_of_rings X) (U : opens X) : comm_ring (F U) :=
 F.Fring U
 
-instance presheaf_of_topological_rings.comm_ring {X : Type*} [topological_space X]
+instance presheaf_of_topological_rings.comm_ring {X : Type u} [topological_space X]
   (F : presheaf_of_topological_rings X) (U : opens X) : comm_ring (F U) :=
 F.Fring U
 
 structure presheaf_of_rings.f_map
-  {X : Type*} [topological_space X] {Y : Type*} [topological_space Y]
+  {X : Type u} [topological_space X] {Y : Type u} [topological_space Y]
   (F : presheaf_of_rings X) (G : presheaf_of_rings Y) :=
```

Comment:
So this is an actual change in our code right? We're going down to one universe. Was this forced upon us by other choices? My instinct would be to use one universe when it was convenient but not worry otherwise. I am still unclear about all of this. From a mathematical perspective it still seems absurd to come up with two topological spaces in different universes -- but these CS people know a whole lot more about writing libraries than us.

Reply:
Aah, right. This change was made because we were surprised that `Huber_pair` depended on *three* universe variables :smiley: